scripts/create-devices-inventory-from-cvp.py

```diff
@@ -81,23 +81,30 @@ def main():
 
     if args.container is None:
         # Get a list of all devices
-        # help(clnt.api.get_inventory)
         inventory = clnt.api.get_inventory()
-        # write all IP addresses from the CVP inventory in the all.txt file
-        out_file = f'{out_dir}/all.txt'
+        # write the IP addresses in a file
+        out_file = f'{out_dir}/all.yml'
         with open(out_file, 'w', encoding="utf8") as out_file:
+            out_file.write("anta_inventory:")
+            out_file.write('\n')
```

Comment:
Why not having `\n` in the previous line ?

Reply:
I will use the model from the inventory module